cloud/ovirt/ovirt_auth.py

```diff
@@ -0,0 +1,231 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+#
+# Copyright (c) 2016 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+try:
+    import ovirtsdk4 as sdk
+except ImportError:
+    pass
```

Comment:
It's actually printed [here](https://github.com/ansible/ansible/pull/17364/commits/0bafd8d70b779e93c0006a66a7e293d10f3a8003#diff-964a81589429c39efce1a9607db47ee6R44)